### PR TITLE
workaround u250 pci reset hang

### PR DESCRIPTION
**Pull request by houlz0507**

### CODE REVIEW

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/mailbox.c [hunk 1]
```diff
@@ -1951,18 +1951,43 @@ static int mailbox_offline(struct platform_device *pdev)
 	 */
 	mailbox_disable_intr_mode(mbx, false);
 #else
-	mailbox_disable_intr_mode(mbx, true);
+	/* stop polling */
```
Comment (maxzhen) [inline src/runtime_src/core/pcie/driver/linux/xocl/subdev/mailbox.c]: this maybe not safe. if someone send out a msg at this moment, timer will be activated, still...

File: src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_drv.c [hunk 2]
```diff
@@ -338,6 +340,17 @@ int xocl_hot_reset(struct xocl_dev *xdev, u32 flag)
 	if (mbret)
 		ret = mbret;
 
+        pci_read_config_word(pdev, PCI_COMMAND, &pci_cmd);
+        pci_cmd &= ~PCI_COMMAND_MASTER;
+	pci_write_config_word(pdev, PCI_COMMAND, pci_cmd);
+
+	xocl_wait_pci_status(XDEV(xdev)->pdev, PCI_COMMAND_MASTER,
```
Comment (maxzhen) [inline src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_drv.c]: bad indentation.

File: src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_drv.c [hunk 3]
```diff
@@ -338,6 +340,17 @@ int xocl_hot_reset(struct xocl_dev *xdev, u32 flag)
...
+			PCI_COMMAND_MASTER);
+
+        pci_read_config_word(pdev, PCI_COMMAND, &pci_cmd);
+        pci_cmd |= PCI_COMMAND_MASTER;
+	pci_write_config_word(pdev, PCI_COMMAND, pci_cmd);
```
Comment (maxzhen) [inline src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_drv.c]: bad indentation.

File: src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_drv.c [hunk 4]
```diff
@@ -338,6 +340,17 @@ int xocl_hot_reset(struct xocl_dev *xdev, u32 flag)
 	if (mbret)
```
Comment (maxzhen) [inline src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_drv.c]: does the comment about 60 seconds still apply?

File: src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/mgmt-core.c [hunk 5]
```diff
@@ -712,11 +714,17 @@ void xclmgmt_mailbox_srv(void *arg, void *data, size_t len,
 		 */
 		ret = (int) xclmgmt_hot_reset(lro);
```
[the remark this answers is not in the document]
Comment (houlz0507) [reply src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/mgmt-core.c]: oops. Failed PPC

**Pull request merged**